lib/manageiq/messaging/client.rb

```diff
@@ -170,9 +170,10 @@ def subscribe_background_job(options)
       # Other options are underlying messaging system specific.
       #
```

Comment:
We might want some documentation here that the incoming `options` are single or Array.

On that note, can we rename `options` to `messages`?  It's a weird name as is.

Reply:
:+1: yeah good call, changed to messages